File: backend/src/main/java/space/davids_digital/vk_gpt_bot/integration/vk/dto/VkUserDto.java
package space.davids_digital.vk_gpt_bot.integration.vk.dto;

import com.fasterxml.jackson.annotation.JsonProperty;

public class VkUserDto {
    @JsonProperty("id")
    public long id;

    @JsonProperty("first_name")
    public String firstName;

    @JsonProperty("last_name")
    public String lastName;

    @JsonProperty("is_closed")
    public boolean isClosed;

    @JsonProperty("can_access_closed")
    public boolean canAccessClosed;

    @JsonProperty("photo_200")
    public String photo200;
}

File: backend/src/main/java/space/davids_digital/sweetie/integration/vk/dto/VkUserDto.kt
package space.davids_digital.sweetie.integration.vk.dto

import com.fasterxml.jackson.annotation.JsonProperty

@Deprecated("Use new API")
open class VkUserDto {
    @JsonProperty("id")
    var id: Long = 0

    @JsonProperty("first_name")
    var firstName: String? = null

    @JsonProperty("last_name")
    var lastName: String? = null

    @JsonProperty("is_closed")
    var isClosed = false

    @JsonProperty("can_access_closed")
    var canAccessClosed = false

    @JsonProperty("photo_200")
    var photo200: String? = null
}
